firebase-config/src/jsTest/kotlin/dev/gitlive/firebase/remoteconfig/FirebaseRemoteConfig.kt

```diff
@@ -9,8 +9,8 @@ actual fun runTest(test: suspend () -> Unit) = GlobalScope
     .promise {
         try {
             test()
-        } catch (e: dynamic) {
-            (e as? Throwable)?.log()
```

Comment:
Throwable maps to Error js so this won't catch other non-errors that might be thrown (I observed the firebase libs throwing non-errors)

Reply:
I'll revert that, thx!

Reply:
@nbransby done